tests/integration/docker_utils/test_docker.py

```diff
@@ -245,6 +245,9 @@ def test_create_with_port_mapping(self, docker_client: ContainerClient, create_c
         ports.add(45180, 80)
         create_container("alpine", ports=ports)
 
+    @pytest.mark.skip(
```

Comment:
@whummer @alexrashed @thrau I skipped this test as it is failing when tests are running docker in docker. Volume mounts typically have to come from the host. This would likely work locally but not in a docker run, thoughts?

Reply:
That's somewhat strange since the tests are currently executed in Docker when building the image: https://github.com/localstack/localstack/blob/436f129ee36e492c9a410db14a36e5381b42e83b/Dockerfile#L98-L101
And since they are executed within a `RUN` command in Docker they should be executed inside an intermediate container already, right?